_posts/2022-02-17-quarkus-superheroes-to-the-rescue.adoc

```diff
@@ -0,0 +1,240 @@
+---
+layout: post
+title: 'Quarkus Superheroes to the Rescue!'
+date: 2022-02-17
+tags: apicurio avro example fault-tolerance help http kafka messaging metrics micrometer mutiny openapi reactive resteasy resteasy-reactive sample testing web wiremock
+synopsis: Introduction to the Quarkus Superheroes sample application, discusses some of the requirements for building it, and illustrates how to run it locally or deploy it to Kubernetes.
+author: edeandrea
+---
+:imagesdir: /assets/images/posts/quarkus-superheroes-to-the-rescue
+:toc:
+
+= Quarkus Superheroes to the Rescue!
+
+image::Duke_Superhero.png[alt=Duke Superhero, align=left, width=50%]
```

Comment:
just to avoid troubles - rename the file and link to use lower case

Reply:
Done - just pushed new commit